samples/wifi/shutdown/README.rst

```diff
@@ -28,6 +28,15 @@ The sample:
 #. Shuts down the Wi-Fi driver.
 #. Puts the nRF5340 SoC in the lowest possible power consumption mode.
 
+Shutdown mode
+*************
+
+The Wi-Fi chipset power state is automatically managed by the Wi-Fi driver, if the network interface is down the Wi-Fi chipset is put in `shutdown` mode.
```

Comment:
This is exactly what we need to have here :) I suggest breaking down in smaller parts and being more verbose. 
My proposal, based on offline info from @krish2718 :

"The nRF70 Series transition to and from Shutdown state is automatically managed by the Wi-Fi driver. When the network interface (net_if: or @krish2718  , you can use an "official" term here which describes the zephyr network interface), is brought down, the Wi-Fi driver puts the nRF70 Series device in `Shutdown` power state. When the network interface is brought up, the Wi-Fi driver puts the nRF70 Series device in `Active` state.